storage/innobase/xtrabackup/test/suites/binlog/ib_slave_info.sh

```diff
@@ -135,15 +136,24 @@ fi
 
 rm -rf $topdir/backup
 
+# Setup a new slave and let it catchup with master data prior to GTID
+start_server_with_id $slave2_id
+setup_slave $slave2_id $master_id
+sync_slave_with_master $slave2_id $master_id
+mysql -e "SHOW SLAVE STATUS\G"
+mysql -e "SELECT * FROM performance_schema.replication_applier_status_by_worker\G"
```

Comment:
we don't have OUT file (like mtr). Did you add them for debugging purpose?

Reply:
fixed